### DISCUSSION TITLE
Possible to use Tasmota as a wifi console server - for out of band management ?

### DISCUSSION THREAD

**Discussion by rapi3** [arendst/Tasmota#19399] [Q&A] [answered]
Hi,
Any idea if it is possible to use Tasmota as a wifi console server, to control remote devices that have serial console for out of band management ( Cisco / HP.... ) ?
At the moment I use/waste a [Raspberry Pi WiFi Console Server](https://www.youtube.com/watch?v=uqq1fWYS18k) to have out of band management for remote devices... I think a wemos d1 with Tasmota will be a better use.
Thank you.

**Comment by rapi3** [comment, marked as answer]
If I am not mistaken implementation it is present in [Serial to TCP Bridge](https://tasmota.github.io/docs/Serial-to-TCP-Bridge/)
and all I need is to figure the hw connection to device console port ?

`When [compiling your build](https://tasmota.github.io/docs/Compile-your-build/) add the following to user_config_override.h:

#ifndef USE_TCP_BRIDGE 
#define USE_TCP_BRIDGE          //  Add support for Serial to TCP bridge (+1.3k code)
#endif`

**Reply by barbudor** [reply]
Yes, check doc here : https://tasmota.github.io/docs/Commands/#tcpbaudrate

Note that these commands are also used with the Modbus Bridge but these are 2 different features

**Reply by rapi3** [reply]
thanks for the confirmation, now I need to get into the soldering state to finish the hw and test the device next week.

**Reply by barbudor** [reply]
Although I love Tasmota and use it for many things, for that kind of project, I'ù using ESP-link : https://github.com/jeelabs/esp-link
It's old and hasn't received any update since a few year but works like a charm.

**Reply by rapi3** [reply]
I succeed to test both Tasmota and esp-link ( esp-link project it is really old and searching for the flash files and info for the wemos/nmcu was a pain in the... back ).
Important thing is both Tasmota and ESP-link work for me to connect on serial console on switch/wlc.
I will see after more nights of tests who will remain in use ( probably Tasmota as it has more security added and it is updated ).

Thank you for hint.
